src/storage.ts

```diff
@@ -779,6 +755,10 @@ export class Storage extends Service {
 
     this.retryOptions = config.retryOptions;
 
+    this.storageTransport = new StorageTransport({...config, ...options});
+    //this.interceptors = [];
+    this.universeDomain = options.universeDomain || DEFAULT_UNIVERSE;
```

Comment:
wheres this change coming from?

Reply:
What change are you referring to? It probably shows all three lines as new because I commented out the interceptors and added StorageTransport. I didn't touch universedomain.